otx/algorithms/detection/tasks/inference.py

```diff
@@ -202,8 +207,9 @@ def _explain_detector(
             dataset=dataset,
             explainer=explain_parameters.explainer if explain_parameters else None,
         )
+        detections = results["outputs"]["detections"]
         explain_results = results["outputs"]["saliency_maps"]
-        return explain_results
+        return detections, explain_results
```

Comment:
Does these return items correspond to the docstring and type annotation of the method?

Reply:
Thanks for the notice, fixed